vnext/Desktop/WinHTTPWebSocketResource.cpp

```diff
@@ -0,0 +1,447 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
```

Comment:
Going to assume this is superseded by WinRTWebSocketResource?

Reply:
Correct.
I will leave it in for experimental purposes until WinRTWsRsc is proven stable, but it won't be linked into the DLL, and likely to be deleted soon.